README.md

```diff
@@ -2,6 +2,12 @@
 
 This addon goal is to make it easy the integration of [bugsnag](https://www.bugsnag.com) to an ember app.
 
+## Compatibility
+
+* Ember.js v3.4 or above
```

Comment:
This is the default output from the current blueprint. I'm not sure what version this addon officially supports. I can change this (and add those versions to travis as well).